vignettes/release-announcement-0-2-0.Rmd

```diff
@@ -0,0 +1,137 @@
+```{r}
+#| include: false
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're excited to announce that the second release of the S7 package is now available on CRAN!
+S7 is a new object-oriented programming (OOP) system designed to succeed both S3 and S4.
+You might wonder why R needs a new OOP system when we already have two.
+The reason lies in the history of R's OOP journey: S3 is a simple and effective system for single dispatch, while S4 adds formal class definitions and multiple dispatch, but at the cost of complexity.
+This has forced developers to choose between the simplicity of S3 and the sophistication of S4.
+
+The goal of S7 is to unify the OOP landscape by building on S3’s existing dispatch system and incorporating the most useful features of S4 (along with some new ones), all with a simpler syntax.
+
+S7’s design and implementation have been a collaborative effort by a working group from the [R Consortium](https://www.r-consortium.org), including representatives from R-Core, Bioconductor, tidyverse/Posit, ROpenSci, and the wider R community.
+Since S7 builds on S3, it is fully compatible with existing S3-based code.
+It's also been thoughtfully designed to work with S4, and as we learn more about the challenges of transitioning from S4 to S7, we’ll continue to add features to ease this process.
+
+Our long-term goal is to include S7 in base R, but for now, you can install it from CRAN:
+
+```{r, eval = FALSE}
+install.packages("S7")
+```
+
+## What’s New in the Second Release
+
+The second release of S7 brings refinements and bug fixes.
+Key new features include:
+
+-   The ability to define classes with lazy property default values.
+-   Initialization logic for properties with custom setters.
+-   Substantial speed improvements for property setting and getting with `@` and `@<-`.
+-   Expanded support for base S3 classes.
+-   `convert()` now has a default method for transforming a parent class instance into a subclass.
+
+Additionally, there are numerous bug fixes and quality-of-life improvements, such as better error messages, improved support for base Ops methods, and compatibility improvements for using `@` in R versions prior to 4.3.
+
```

Comment:
Maybe add a section here for "who should use S7"?

A few thoughts:

* There are two packages on CRAN already using it.
* The tidyverse team is using it for new packages
* You probably still need to be generally an earlier adopter of new technology; but you're definitely not the first, so why you might run in to some new problems, many issues have already been discovered and fixed.